src/bundle/makeRepairDataProperties.js

```diff
@@ -4,19 +4,17 @@
 // https://github.com/google/caja/blob/master/src/com/google/caja/ses/repairES5.js
 
 export default function makeRepairDataProperties() {
```

Comment:
Why does this export a `makeRepairDataProperties` rather than just defining and exporting `repairDataProperties`? If each `repairDataProperties` were stateful, or might be stateful in the future, that would make sense. But I don't see it here.

Reply:
Fixed.